testing/src/main/java/com/yubico/yubikit/testing/piv/PivTestState.java

```diff
@@ -16,15 +16,151 @@
 
 package com.yubico.yubikit.testing.piv;
 
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assume.assumeFalse;
+import static org.junit.Assume.assumeTrue;
+
+import com.yubico.yubikit.core.YubiKeyDevice;
+import com.yubico.yubikit.core.application.ApplicationNotAvailableException;
+import com.yubico.yubikit.core.smartcard.ApduException;
+import com.yubico.yubikit.core.smartcard.SmartCardConnection;
+import com.yubico.yubikit.management.Capability;
+import com.yubico.yubikit.management.DeviceInfo;
 import com.yubico.yubikit.piv.KeyType;
+import com.yubico.yubikit.piv.ManagementKeyType;
+import com.yubico.yubikit.piv.PivSession;
+import com.yubico.yubikit.testing.ScpParameters;
+import com.yubico.yubikit.testing.TestState;
+
+import java.io.IOException;
+
+import javax.annotation.Nullable;
+
+public class PivTestState extends TestState {
+
+    static final char[] DEFAULT_PIN = "123456".toCharArray();
+    static final char[] DEFAULT_PUK = "12345678".toCharArray();
+    static final byte[] DEFAULT_MANAGEMENT_KEY = new byte[]{
+            0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08,
+            0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08,
+            0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08,
+    };
+
+    private static final char[] COMPLEX_PIN = "11234567".toCharArray();
+    private static final char[] COMPLEX_PUK = "11234567".toCharArray();
+    private static final byte[] COMPLEX_MANAGEMENT_KEY = new byte[]{
+            0x01, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07,
+            0x01, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07,
+            0x01, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07,
+    };
+
+    public final boolean isFipsApproved;
+    public char[] defaultPin;
+    public char[] defaultPuk;
+    public byte[] defaultManagementKey;
```

Comment:
These should probably be renamed as they aren't always the default values. Maybe "initialPin" or similar?

Reply:
fixed!